stories/topics/con-gcp-application-architecture.adoc

```diff
@@ -10,18 +10,9 @@ image::aap-on-gcp-architecture.png[GCP application architecture]
-{PlatformNameShort} components that are deployed are isolated into their own VPC. 
-Applications then run on instances managed by an instance group, which run on RHEL-based Google Compute machine images. 
-These machine images are preloaded with required container images to run {PlatformNameShort} including {Hubname}, {ControllerName}, and {ReceptorRpm} components. 
-A shared Google Filestore volume is mounted into each instance deployed by the instance group manager.
-
-All GCP Compute instances have private IP addresses by default. 
-The {PlatformNameShort} applications are run in containers managed through podman on the instances. 
-This podman configuration is itself managed by systemd. 
-SELinux is enabled on GCP Compute instances and is supported down to the container level. 
-Each {ControllerName} instance is configured as an {PlatformNameShort} hybrid node by default to simplify the scale up and out procedures.
+{AAPonGCP} is designed to be private, with no public access allowed by default. This requires customers to expose the deployed internal Load Balancers (ILBs) themselves pursuant to their own network requirements and security practices (some potential ways to expose the ILBs would include VPC Peering, Transit Gateway, VPN, External Load Balancers, etc...). All cloud infrastructure components are deployed in a Virtual Private Cloud (VPC). Customers may choose between deploying into an existing VPC or they may opt to have the product deploy a new VPC for them.  All VM instances and Cloud infrastructure have private IP addresses (allocation determined by the VPC and subnetworks specified at deployment time) by default. All internal traffic is encrypted using self-signed certificates generated at deployment time (external traffic may also be encrypted by deploying your own certificate on the Internal Load Balancers deployed by the product). The {PlatformNameShort} software runs as containers on the deployed VM instances.  VM instance lifecycles are managed by managed instance groups (MIGs) that monitor the health of each service running on the VM instances and will automatically cycle the VM instances down and replace them with new VM instances if the health check fails to respond ensuring that the {PlatformNameShort} services stay up and available to process requests. The VM instances run a customized RHEL (RedHat Enterprise Linux) Google Cloud Machine Image as their base image. This Google Cloud Machine Image is preloaded with all the required container images and packages to run the {PlatformNameShort} ({HubName}, {ControllerName}, and Execution Node components). A shared Google File Store (GFS) volume is mounted into each VM instance provisioned by the product and is utilized for shared access to common files and resources.  A Google Cloud SQL Service is provisioned by the product at deployment time and contains databases for both the {ControllerName} and {HubName}.
```

Comment:
I(i)nternal Load Balancers ? (wrap in _)
some potential ways to expose the ILBs include ...  and External Load Balancers, amongs others.
Customer can choose...  or can choose to have the product deploy....
time. You can also encrypt external traffic by deploying ....
Managed Instance Groups (wrap in _) manage VM instance lifecycles that monitor ... and automatically cycle
respond, ensuring that